rainloop/lib/Controller/PageController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace OCA\RainLoop\Controller;
+
+use OCA\RainLoop\Util\RainLoopHelper;
+
+use OCP\App\IAppManager;
+use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http\ContentSecurityPolicy;
+use OCP\AppFramework\Http\TemplateResponse;
+use OCP\IConfig;
+use OCP\IRequest;
+
+class PageController extends Controller {
+	private $userId;
+	private $config;
+	private $appManager;
+	private $appPath;
+
+	public function __construct($AppName, IRequest $request, IAppManager $appManager, IConfig $config) {
+		parent::__construct($AppName, $request);
+		$this->appPath = $appManager->getAppPath('rainloop');
+		$this->config = $config;
+		$this->appManager = $appManager;
+	}
+
+	/**
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 */
+	public function index() {
```

Comment:
You don't see it, but if you bring up the developer tools in your browser and looks at the response to `apps/rainloop/` (the root URL), you see:

```
This application requires JavaScript for correct operation. Please enable JavaScript and reload the page.
Skip to main content Skip to navigation of app
```

The user never sees that, however. I'm unsure at the moment why that is happening, or even if it matters at all. At first I was sure this was something I had introduced, but I brought up a clean install of NC13 with a the rainloop app as represented by the branch `nc13` and the error is still there, so apparently this was there before.